src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -56,6 +56,120 @@ private TexlParser(IReadOnlyList<Token> tokens, Flags flags)
             _flags = flags;
         }
 
+        public static ParseUDFsResult ParseUDFsScript(string script, CultureInfo loc = null)
+        {
+            Contracts.AssertValue(script);
+            Contracts.AssertValueOrNull(loc);
+
+            var formulaTokens = TokenizeScript(script, loc, Flags.NamedFormulas);
+            var parser = new TexlParser(formulaTokens, Flags.NamedFormulas);
+
+            return parser.ParseUDFs(script);
+        }
+
+        private ParseUDFsResult ParseUDFs(string script)
+        {
+            var udfs = new List<UDF>();
+            ParseTrivia();
```

Comment:
Not needed since the first line of the while loop is also ParseTrivia

Reply:
Fixed